src/custom-types.json

```diff
@@ -140,6 +140,33 @@
   "application/vnd.android.package-archive": {
     "compressible": false
   },
+  "application/vnd.apple.pages": {
+    "compressible": false,
+    "extensions": ["pages"],
+    "sources":[
+      "https://codex.wordpress.org/Function_Reference/get_allowed_mime_types",
+      "https://docs.alfresco.com/5.2/references/valid-transformations.html",
+      "https://tika.apache.org/1.15/api/org/apache/tika/parser/iwork/IWorkPackageParser.html",
+    ]
+  },
+   "application/vnd.apple.numbers": {
+    "compressible": false,
+     "extensions": ["numbers"],
+     "sources":[
+       "https://codex.wordpress.org/Function_Reference/get_allowed_mime_types",
+       "https://docs.alfresco.com/5.2/references/valid-transformations.html",
+       "https://tika.apache.org/1.15/api/org/apache/tika/parser/iwork/IWorkPackageParser.html",
```

Comment:
None of these are a primary reference. Who owns this file extension? I believe Apple, right? Please like to something Apple controls.